### PR TITLE
Da/patch lambda directly

### PR DESCRIPTION
**Pull request by danamansana**

### CODE REVIEW

File: src/helpers/ApiHelper.js [hunk 1]
```diff
@@ -317,6 +326,45 @@ const ApiHelper = {
...
+          item.error = errorResponse;
+          // Handle retries or fatal errors by error status code
+          return errorHandlerFn(errorResponse, 'recap-service', item, callback);
+        })
+      }
+
+      // Skip item if checkoutProccessed or checkinProccessed is not true
+      logger.info(`Will not patch Cancel Request Record: (${item.id}) in Recap. checkoutProccessed is: ${item.checkoutProccessed} and checkinProccessed is ${item.checkinProccessed}`);
+      return callback(null, item);
+    }
```
Comment (nonword) [inline src/helpers/ApiHelper.js]: There sure is a lot of mixing of callbacks and promises in this app, which seems a shame, but does seem to be the style chosen.

File: src/helpers/ApiHelper.js [hunk 2]
```diff
@@ -249,6 +256,7 @@ const ApiHelper = {
     });
   },
   postCheckOutItem (apiUrl, token, errorHandlerFn, item, callback) {
+    logger.info('checking out');
```
Comment (nonword) [inline src/helpers/ApiHelper.js]: should this indicate what it's checking out, or are the batches of items small enough that it's clear in context?

File: src/helpers/ApiHelper.js (hunk 1 above)
Comment (danamansana) [reply src/helpers/ApiHelper.js]: Totally agree

File: src/helpers/ApiHelper.js (hunk 2 above)
Comment (danamansana) [reply src/helpers/ApiHelper.js]: Just one at a time, but I think it's probably a good idea to add some indication of which item is being checked out. I'll add that in.

**Pull request merged**